backend/prisma/schema.prisma

```diff
@@ -69,7 +70,40 @@ model Plan {
   specialEventType EventType
   createdAt        DateTime         @default(now())
   updatedAt        DateTime         @updatedAt
-  PlanMembership   PlanMembership[]
+  planMembership   PlanMembership[]
+  contact          Contact          @relation(fields: [contactID], references: [id])
+  contactID        String
```

Comment:
can you make this ContactID `String?` for not required.

Reply:
done